src/libs/Navigation/AppNavigator/createCustomFullScreenNavigator/index.tsx

```diff
@@ -0,0 +1,83 @@
+import type {ParamListBase, StackActionHelpers, StackNavigationState} from '@react-navigation/native';
+import {createNavigatorFactory, useNavigationBuilder} from '@react-navigation/native';
+import type {StackNavigationEventMap, StackNavigationOptions} from '@react-navigation/stack';
+import {StackView} from '@react-navigation/stack';
+import React, {useEffect, useMemo} from 'react';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import navigationRef from '@libs/Navigation/navigationRef';
+import SCREENS from '@src/SCREENS';
+import CustomFullScreenRouter from './CustomFullScreenRouter';
+import type {FullScreenNavigatorProps, FullScreenNavigatorRouterOptions} from './types';
+
+// TODO-IDEAL: Extract to utils with ./createCustomStackNavigator/index.tsx
```

Comment:
We should avoid leaving comments like this. Let's create an issue for it instead.